### ISSUE TITLE
Problem with canproco image `sub-mon118` in PSIR contrast

### ISSUE THREAD

**Issue by plbenveniste** [neuropoly/data-management#262]
The file is : `sub-mon118_ses-M0_PSIR.nii.gz`
In the PSIR image from subject `sub-mon118`we can see the following slices. 

![ezgif com-resize (7)](https://github.com/neuropoly/data-management/assets/67429280/1db41285-358a-441e-8085-2ac9b6dd585c)

It seems that something strange happened at slice 6 which presents opposite contrast to the other slices. 
It is as if every slices are from the PSIR contrast except for slice 6 which appears similar to STIR.

What should be done ?

**Comment by valosekj** [comment]
Good catch! It looks like a strong motion artifact. We should exclude this image. There is an [exclude.yml](https://github.com/ivadomed/canproco/blob/main/etc/exclude.yml) file in the canproco repo for this purpose.

**Comment by jcohenadad** [comment]
For the canproco project, I suggest to report these in https://github.com/ivadomed/canproco/issues with `data-quality` label, so Shannon and her team is aware (this current repos is private).
